### DISCUSSION TITLE
[Projects Beta] View full issues via side pop-out

### DISCUSSION THREAD

**Discussion by davidjb99** [community/community#9927] [Projects and Issues]
Presently you can't view issues in Projects Beta using the same draft pop out view, forcing you to open a new windows to view the issue. 

It would be far more constant and much nicer UX to be able to view a full issue via the sidebar pop out like you can with a draft

**Comment by steebchen** [comment]
This is a duplicate of https://github.com/github/feedback/discussions/5381.
